central/cve/fetcher/istio.go

```diff
@@ -67,7 +68,13 @@ func (m *istioCVEManager) setCVEs(cves []*storage.EmbeddedVulnerability, nvdCVEs
 	m.embeddedCVEs = cves
 }
 
-func (m *istioCVEManager) updateCVEs(newCVEs []*schema.NVDCVEFeedJSON10DefCVEItem) error {
+func (m *istioCVEManager) updateCVEs(newCVEs []*schema.NVDCVEFeedJSON10DefCVEItem) (retErr error) {
+	defer func() {
+		if r := recover(); r != nil {
```

Comment:
Note that this does not catch a `panic(nil)` (and doing so is possible, but definitely a bit of a pain). `safe.RunE` handles this correctly. (I wouldn't be saying this if I hadn't once wasted hours hunting a mysterious API failure with no trace in the logs but still completely bogus results, until I discovered that a `panic(nil)` was just swallowed because of code like this)